playbooks/byo/openshift-etcd/config.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: Create initial host groups for localhost
```

Comment:
Instead of putting these two tasks in this playbook, please include the byo/openshift-cluster/initialize_groups.yml.  This standardizes the tasks, and also flags this playbook as an entry point, which then will run CI syntax checks.